R/dash.R

```diff
@@ -268,7 +268,14 @@ Dash <- R6::R6Class(
         TRUE
       })
 
-
+      route$add_handler('get', '/*', function(request, response, keys, ...) {
```

Comment:
Did you have to move this around so that it gets declared _after_ the other routes or does it work like this?

Reply:
Oh nvm, the original route wasn't removed: https://github.com/plotly/dashR/pull/53/files#diff-1881af3720f8f49fd3a44e51c9fb5a0dR180

Let's 🔪 the original route since `/*` should catch `/`. 

Also, this route needs to be prefixed by `routes_pathname_prefix`

Reply:
Right, I noticed that when I glanced at the Python code but forgot to insert here.

Reply:
fixed in 62d7dd9965e8beb42002031830c6c30c377fcb1f